### PR TITLE
Remove dnxcore50 moniker from lock file

### LINKED ISSUE #2810: Remove dnxcore50 moniker from lock file
That thing leaks everywhere and is not used anymore.

### PR DESCRIPTION
**Pull request by forki**
fixes #2810

### CODE REVIEW

File: src/Paket.Core/Dependencies/NuGetCache.fs [hunk 1]
```diff
@@ -176,7 +179,8 @@ let tryGetDetailsFromCache force nugetURL (packageName:PackageName) (version:Sem
                 if (PackageName cachedObject.PackageName <> packageName) ||
                     (cachedObject.Version <> version.Normalize())
                 then
-                    traceVerbose (sprintf "Invalidating Cache '%s:%s' <> '%s:%s'" cachedObject.PackageName cachedObject.Version packageName.Name (version.Normalize()))
+                    if verbose then
```
Comment (matthid) [inline src/Paket.Core/Dependencies/NuGetCache.fs]: macht den check nicht auch traceVerbose schon?

Comment (forki) [reply src/Paket.Core/Dependencies/NuGetCache.fs]: yes, but the right side (sprintf ...) would be evaluated even if verbose is false.
This was serious perf problem in the past

**Comment by forki** [comment]
if we move it into extractPlatforms then we get the annoying warnings that dnxcore is not real moniker

File: src/Paket.Core/Dependencies/NuGetCache.fs (hunk 1 above)
Comment (matthid) [reply src/Paket.Core/Dependencies/NuGetCache.fs]: Should we remove `traceVerbose` in that case ?

**Comment by forki** [comment]
I did that a while ago, but I'm not remembering  why it didn't work fully

Am 02.10.2017 12:21 schrieb "Matthias Dittrich" <notifications@github.com>:

**Comment by matthid** [comment]
Tests need to be adapted. The current failure looks a bit suspicious, but it might be that the error message is just incomplete and everything is fine.

**Comment by forki** [comment]
Working on the tests. Will come later today

Am 02.10.2017 12:24 schrieb "Matthias Dittrich" <notifications@github.com>:

**Pull request merged**